docs/UserGuide.md

```diff
@@ -837,10 +837,92 @@ any bugs or have any suggestions on the application, please go to
 <a href="https://github.com/AY2021S1-CS2103T-F12-2/tp/issues">our github issue page</a> to 
 issue the bugs and the suggestions. Great appreciation from the development team of StudyBananas.
 
-## 8. Command summary
+## 8. Summary
+This section summarises all available commands in StudyBananas. 
+
+### 8.1. Summary of Commands (Petrick)
```

Comment:
![image](https://user-images.githubusercontent.com/25611768/98536013-081c3c00-22c2-11eb-90b7-23daecae9d07.png)

There seems to be an error for this table parsing. I suggest using the default markdown table instead: https://www.tablesgenerator.com/markdown_tables#.